### ISSUE TITLE
color of the error-message in the message view

### ISSUE THREAD

**Issue by jkapferer** [microsoft/vscode-mssql#17725]
Hi there

I use this vs code extension every day and I am very happy with it. I have choosesn a dark mode in VS Code.

But as I am completely colorblind I can hardly read the red error-messages in the message view based on a black background.

It would be very kind, if you could change the color of even make it parameterable.

Many thank's for your works.

Kind regards
Jürg Kapferer

![messageView_20230627](https://github.com/microsoft/vscode-mssql/assets/44819593/8c532f6c-6c2d-4433-afc8-ee09159ad4a9)

**Comment by maiklindner** [comment]
I'm suffering from the same problem. I'm not entirely sure but that seems to be a result of here #1068 - Thanks Ian ;)

**Comment by Benjin** [comment]
We rewrote the query results pane this past winter/spring, and it now uses the error color for your currently-selected VS Code theme.  If you're still having issues and it's not an issue with the theme's color choices, please reopen this bug!
